hopsanc/hopsanc.pro

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------
+# Global project options
+# -------------------------------------------------
+include( ../Common.prf )
+
+TARGET = hopsanc
+TEMPLATE = lib
+CONFIG += shared
+DESTDIR = $${PWD}/../bin
+TARGET = $${TARGET}$${DEBUG_EXT}
+
+QT -= core gui
+
+# Enable C++11
+lessThan(QT_MAJOR_VERSION, 5){
+  QMAKE_CXXFLAGS += -std=c++11
+} else {
+  CONFIG += c++11
+}
+
+
+#--------------------------------------------------
+# Add the include path to our self, (HopsanC)
+INCLUDEPATH *= $${PWD}/include/
+#--------------------------------------------------
+
+#--------------------------------------------------
+# Add the include path to (HopsanCore)
+INCLUDEPATH *= $${PWD}/../HopsanCore/include/
+LIBS *= -L$${PWD}/../bin -lhopsancore$${DEBUG_EXT}
+#--------------------------------------------------
+
+# -------------------------------------------------
+# Platform specific additional project options
+# -------------------------------------------------
+win32 {
+    DEFINES += HOPSANGENERATOR_DLLEXPORT
+    DEFINES -= UNICODE
+}
+unix {
+     # Add runtime search path so that dynamically loaded libraries in the same directory can be found.
+    # Note! QMAKE_LFLAGS_RPATH and QMAKE_RPATHDIR does not seem hande $$ORIGIN, adding manually to LFLAGS
+    QMAKE_LFLAGS *= -Wl,-rpath,\'\$$ORIGIN/./\'
+}
+
+# -------------------------------------------------
+# Project files
+# -------------------------------------------------
+HEADERS = \
+    include/hopsanc.h \
```

Comment:
I think that header files should be "namespaced" with an additional sub-directory with he name of this library, to avoid name collision. For instance hopsanc_win32dll.h could then still be called win32dll.h
I know it feels silly when there  are so few files, but you avoid trouble in the future by always namespacing include files.

We should do this for new libraries